cross/cryptography-legacy/Makefile

```diff
@@ -0,0 +1,14 @@
+PKG_NAME = cryptography
+PKG_VERS = 3.3.2
```

Comment:
3.4.8 is the last version you can probably compile for legacy, with CRYPTOGRAPHY_DONT_BUILD_RUST.

idk if you want to update that package now so you don't have to touch this part again or if you want to leave it for another time.

Reply:
Thnx for the info but cryptography-legacy is really only meant for not breaking `python2` and `python38`, both of which are expected to be removed shortly.

Reply:
Only concern I would have, is some projects pin 3.4.8 (34.8) to make their packages continue to work on alternative hardware.

Reply:
And everyone as the possibility to add it to their own `requirement-crossenv.txt` file as they see fit :)